src/snackbar/types.ts

```diff
@@ -7,21 +7,15 @@ export interface SnackbarOptions {
     autoHideDuration?: number | null;
 }
 
-export interface SnackbarState {
+export interface SnackbarState extends SnackbarOptions {
+    success: (message: string, options?: Partial<SnackbarOptions>) => void;
```

Comment:
Now that we are at it, can we create a type `(message: string, options?: Partial<SnackbarOptions>) => void` to DRY the interface?